app/my_main.py

```diff
@@ -0,0 +1,50 @@
+
+from app.knights.knight import Knight
+from app.armours.armour import Armour
+from app.weapons.weapon import Weapon
+from app.weapons.potions.potion import Potion
+from app.battle.battle import Battle
+
+
+lancelot = Knight('lancelot', 35, 100)
```

Comment:
Lots of magic const, Use KNIGHTS dictionary from main.py

Reply:
Thanks for the comment, but they are needed for logical implementation!